### DISCUSSION TITLE
Stuck on ":: ℹ selected series: 3060ti" after running the program

### DISCUSSION THREAD

**Discussion by Rafa0550** [jef/streetmerchant#1761] [Help]
I finished editing the dotenv file and when i do npm run start, this is what happens.

C:\streetmerchant>npm run start

> streetmerchant@ start C:\streetmerchant
> npm run compile && node build/src/index.js

> compile
> tsc

ＳＴＲＥＥＴＭＥＲＣＨＡＮＴ
3.4.1

[1:47:08 AM] info :: ℹ selected stores: bestbuy, newegg, amazon, bandh, zotac
[1:47:08 AM] info :: ℹ selected models: 3060ti
[1:47:08 AM] info :: ℹ selected series: 3060ti

And it stays like this.

**Comment by wofnull** [comment]
remove the 3060ti from the models selection, as this is not a model.

for reference look at https://github.com/jef/streetmerchant/blob/main/docs/reference/filter.md for supported models.
